packages/expo-contacts/src/Contacts.ts

```diff
@@ -943,6 +943,17 @@ export async function getGroupsAsync(groupQuery: GroupQuery): Promise<Group[]> {
   return await ExpoContacts.getGroupsAsync(groupQuery);
 }
 
+/**
+ * Presents a native contact picker to select a single contact from the system.
+ * @return A promise that fulfills with a `Contact` object if a contact was selected, or `null` if no contact was selected (picking was cancelled).
```

Comment:
@amandeepmittal On Android, for this to work, contact permissions need to be granted first, but on iOS this is not necessary (permissions are handled automatically by iOS). What is the proper way/syntax to highlight this in the docs?

Reply:
Hey @fobos531, you can describe this in the method definition both for Android and iOS. When adding the description, for Android, you can link to [`requestPermissionAsync()`](https://docs.expo.dev/versions/latest/sdk/contacts/#contactsrequestpermissionsasync) since that is used to ask for user permission. This will help folks to be aware that they need to call this function in their app code for Android.